### PR TITLE
fix SpaceFN

### PR DESCRIPTION
**Pull request by MyController**

### CODE REVIEW

File: public/json/spacefn.json [hunk 1]
```diff
@@ -507,7 +507,12 @@
               "value": 0
             }
           ]
-        },
+        }
+      ]
+    },
+    {
+      "description": "SpaceFN: Space+e to Escape, Space+m to Backtick (`), Space+Comma to Tilde (~)",
```
Comment (MuhammedZakir) [inline public/json/spacefn.json:514]: I recommend to also split `space+e -> escape` to another rule so users can choose between `space+e` and ``space+` `` for `escape` (or both if wanted).

Comment (MuhammedZakir) [reply public/json/spacefn.json:519]: If this is intended, please tell, so I can merge this.

Comment (MyController) [reply public/json/spacefn.json:519]: Yes, please merge it

**Comment by MuhammedZakir** [comment]
Done. Thanks! :-)

**Pull request merged**